src/coffee/run.coffee

```diff
@@ -35,7 +35,7 @@ argv = require('optimist')
   .describe('logDir', 'directory to store logs')
   .describe('logSilent', 'use console to print messages')
   .describe('timeout', 'Set timeout for requests')
-  .default('csvHeaders', 'sku, quantity')
+  .default('csvHeaders', 'sku, quantityOnStock')
```

Comment:
No, we are not supporting custom headers again.
We using the API key names as a guide.
That removes some level of complexity and mappings